CirrusRedOrg/EntityFrameworkCore.Jet
# Misc test fixes

- Remove tests for triggers as not applicable for Jet
- FromSqlQueryJetTest
  - Update its infrastructure
  - Override some tests as the raw sql needed a modification. These tests needed a query that referenced a table so used the Dual table that is set up for this purpose
 - Update Gears of War related tests to fix up the Timeline value in the Missions table. Jet can't handle datetime values before the year 100 so adjust the seed and expected values accordingly
 - Fix up some stored procedure calls with parameters. Parameters to the `EXEC` do not used the @ sign so picking up and matching was failing and removing all the parameters.
 - Protect the conversion functions from NULL values.. While other databases (e.g. SQL Server) can cast/convert a NULL to a numerical value and it will propagate the NULL, Jet conversion functions CLNG, CSNG etc do not
 - Some other minor fixes
